spec/services/princeton_users_spec.rb

```diff
@@ -100,32 +100,38 @@
     end
   end
 
-  describe "#user_match?" do
+  describe "#user_list_query" do
     let(:user) { { uid: "sms98", name: "Sotomayor, Sonia" } }
-    let(:user_no_name) { { uid: "sms98" } }
+    let(:user_no_name) { { uid: "sms99", name: "" } }
```

Comment:
@carolyncole we have users with a nil name in the database, that's why I had the name as nil in the tests. I am not sure how the new code will handle them.